sdk/src/Core/Amazon.Runtime/Credentials/ProcessAWSCredentials.cs

```diff
@@ -154,48 +158,52 @@ private CredentialsRefreshState SetCredentialsRefreshState(ProcessExecutionResul
             // The system will attempt to find the executable within folders specified by the PATH environment variable.
             if (processInfo.ExitCode == 0)
             {
-                JsonData data = null;
+                JsonDocument doc = null;
```

Comment:
Doesn't the C# version we use in V4 allow us to use `using JsonDocument doc = null`? Then you wouldn't need to call `Dispose` manually every time.

It's available in C# 8.0 (https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide00630), and we're using C# 9.0 (the same comment applies to the other `doc?.Dispose()` occurrences).